src/subscribe.rs

```diff
@@ -0,0 +1,117 @@
+struct Subscriber {
+    name: Option<String>,
+    email: String,
+    phone: Option<String>,
```

Comment:
Also add optional `source` and optional `topic`. In the luma import script you will find how we store `source` and `topic`.

Reply:
this struct holds subscriber info. The `source` and `topic` are added to the subscription below in the handler code.